### ISSUE TITLE
Flatpak installation failed

### ISSUE THREAD

**Issue by dan21dan** [cryinkfly/Autodesk-Fusion-360-for-Linux#382]
**Describe the bug**
I got message "Installer is not well configured." during installation

**To Reproduce**
1. Choose Autodesk Fusion 360 in "bottles" and try to install it
2. After a moment you get message "Installer is not well configured"

**Expected behavior**
No error during installation

**Additional context**
I see in console:
```
19:15:21 (WARNING) Failed to download [[http://www.autodesk.com:80/products/fusion-360/appstream](http://www.autodesk.com/products/fusion-360/appstream)] with code: 403 != 200
19:15:21 (ERROR) Failed to download Fusion 360 Admin Install.exe, or checksum failed.
```

**Comment by alextrical** [comment]
It looks like fusion 360 have removed the Admin install file we used for Fusion 360, I also don't believe this issue is specific to the Flatpak version

**Comment by dan21dan** [comment]
Thanks for connecting me to #380, it's the same case. Looking at the changes, I tried it approximately 3 hours after the `install.sh` script was modified. My guess: changed filename in the `install.sh` script probably didn't get integrated into the installer available in Bottles.

And where did I find it?
I ran Bottles, created a new one, clicked on "Install application" and selected "Autodesk Fusion 360" for installation.
